onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -2462,6 +2484,61 @@ TensorrtExecutionProvider::GetCapability(const GraphViewer& graph,
   return result;
 }
 
+/**
+ * Refit the weight-stripped engine
+ */
+common::Status TensorrtExecutionProvider::RefitEngine(std::string onnx_model_filename,
+                                                      std::string& onnx_model_folder_path,
+                                                      std::string& weight_stripped_engine_cath_path,
+                                                      bool path_check,
```

Comment:
Any reason `path_check` would be set to false? Shouldn't we always enforce relative paths?